syntaxes/v.tmLanguage.json

```diff
@@ -261,30 +263,26 @@
 							}
 						},
 						{
-							"match": "(assert)(?=\\s*\\()",
+							"match": "(fn)(?=\\s*\\()",
 							"captures": {
 								"1": {
-									"name": "keyword.assert.v"
+									"name": "keyword.fn.v"
 								}
-							}
+							}	
 						},
 						{
-							"match": "(sizeof)(?=\\s*\\()",
+							"match": "(assert)(?=\\s*\\()",
 							"captures": {
 								"1": {
-									"name": "keyword.sizeof.v"
+									"name": "keyword.assert.v"
```

Comment:
You can simplify patterns like this by using `"name": "keyword.$1.v"`, where `"$1"` refers to whatever was matched in the `$n`th capturing group:

~~~json
{
	"match": "(assert|in|or|match|if|else|for|go|goto|defer|return|sizeof)(?=\\s*\\()",
	"name": "keyword.control.$1.v"
}
~~~

So `defer` would be scoped with `keyword.control.defer.v`, for example.

Reply:
Oh, this really helpful! Thanks!